Resources/Prototypes/_LostParadise/Entities/Clothing/Uniforms/armbands.yml

```
@@ -0,0 +1,115 @@
+    size: Tiny
+  - type: Butcherable
+    butcheringType: Knife
+    spawned:
+    - id: MaterialCloth1
+      amount: 1
+
+- type: entity
+  parent: LPPArmband
+  id: LPPArmbandRed
+  name: red armband
+  description: 
```

yamllint

[error] trailing spaces

(trailing-spaces)